packages/react-native/src/theme/types.ts

```diff
@@ -17,6 +17,8 @@ import {
   TextFieldStyles,
 } from '../primitives';
 
+type OutputVariantKey = 'input' | 'output' | unknown;
```

Comment:
Think we already have a `OutputVariantKey` in the UI package with a differing defintion. To prevent confusion maybe we can just inline this type on ln 31?